fury/text_tools.py

```diff
@@ -0,0 +1,178 @@
+import fury
```

Comment:
In  the future we need create more stuff to deal with text rendering. Like signed distance fields  MDSF, Unicode characters etc. I think is better to have a specific to deal with text stuff. But I don't know if `text_tools.py` is good name for that.